src/app/src/test/kotlin/org/imperial/mrc/hint/helpers/JSONValidator.kt

```diff
@@ -72,7 +75,9 @@ class JSONValidator {
             "$name.schema.json"
         }
         val url = URL("https://raw.githubusercontent.com/mrc-ide/hintr/$hintrVersion/inst/schema/$path")
-        return url.openStream().use {
+
+        val conn = url.openConnection() as HttpURLConnection
+        return BufferedReader(InputStreamReader(conn.getInputStream())).use {
```

Comment:
Holy hell, this was annoying. This randomly started throwing errors when running locally, when attempting to validate the blameless ValidateInputResponse schema. Eventually worked out it appeared to always fail at the same character index, so presumably it wasn't managing to read the whole file over a certain length). Changing from url.openStream to url.openConnection seemed to fix this...